src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -581,9 +581,6 @@
   <data name="ConverterForPropertyMustBeValid" xml:space="preserve">
     <value>The generic type of the converter for property '{0}.{1}' must match with the specified converter type '{2}'. The converter must not be 'null'.</value>
   </data>
-  <data name="BuiltInConvertersNotRooted" xml:space="preserve">
```

Comment:
No longer needed -- built-in converter factories only consulted from specific entrypoints that are guaranted to root them.